packages/protvista-track/src/protvista-track.js

```diff
@@ -20,24 +20,32 @@ class ProtvistaTrack extends ProtvistaZoomable {
 
   connectedCallback() {
     super.connectedCallback();
-    this._tooltipEvent = this.getAttribute("tooltip-event")
-      ? this.getAttribute("tooltip-event")
-      : "mouseover";
+    this._highlightEvent = this.getAttribute("highlight-event")
+      ? this.getAttribute("highlight-event")
+      : "onclick";
     this._color = this.getAttribute("color");
     this._shape = this.getAttribute("shape");
     this._featureShape = new FeatureShape();
     this._layoutObj = this.getLayout();
     this._config = new ConfigHelper(config);
-    this.createTooltip = this.createTooltip.bind(this);
 
     if (this._data) this._createTrack();
 
+    this._resetEventHandler = this._resetEventHandler.bind(this);
+
     this.addEventListener("load", e => {
       if (_includes(this.children, e.target)) {
         this.data = e.detail.payload;
       }
     });
+    document.addEventListener("click", this._resetEventHandler);
```

Comment:
does it need to be attached to the `document`?

Reply:
OK I got it. this is a click anywhere in the page to close the tooltip. So it makes sense that the event is attached in `document`